operator/cmd/mesh/operator-dump.go

```diff
@@ -28,12 +28,7 @@ type operatorDumpArgs struct {
 
 func addOperatorDumpFlags(cmd *cobra.Command, args *operatorDumpArgs) {
 	hub, tag := buildversion.DockerInfo.Hub, buildversion.DockerInfo.Tag
-	if hub == "" {
```

Comment:
imo we should only do this for development builds (for all instructions) - and permit a HUB environment variable override.

if the release is stable, this conditional code should not be executed.

Reply:
> hub, tag := buildversion.DockerInfo.Hub, buildversion.DockerInfo.Tag

AFAIK this code always returns `docker.io/istio`.

> if the release is stable, this conditional code should not be executed.

Hence `if hub == "" {` never executed.

Reply:
This part might still be needed for dev builds